rest-api/offline/metrics_config.py

```diff
@@ -146,7 +146,7 @@ def _get_answer_values_func(question_code_value):
 
 # Fields generated by our metrics export.
 BASE_PARTICIPANT_FIELDS = ['participant_id', 'date_of_birth', 'first_order_date',
-                           'first_samples_arrived_date', 'first_physical_measurements_date',
+                           'first_samples_arrived_date', 'final_physical_measurements_date',
```

Comment:
Let's still call this "first_physical_measurements_date" -- we're still finding the MIN value. What's changed is we are using the timestamp of the PM (stored as "finalized", parsed from the JSON resource) instead of the server time at which the PM was created (stored as "created"). But either way, it's the *first* PM date for this user.

Reply:
done